.github/workflows/cicd.yml

```diff
@@ -128,11 +143,23 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     needs: [build]
+    environment: prod
     steps:
       - name: 🔽 download packages
         uses: actions/download-artifact@v3
         with:
           name: packages
       - name: Display structure of downloaded files
         run: ls -R
-      # TODO: Publish to NuGet
\ No newline at end of file
+      - name: push
+        run: dotnet nuget push **/*.nupkg  --api-key ${{ secrets.NUGET_API_KEY }} --source "nuget.org" --skip-duplicate
+      - name: 🤘 checkout
```

Comment:
Please move the tagging ro a separate job so that we qould `re-run` nuget publication alone if required.

Reply:
done